lib/data/external/PfsClient.js

```diff
@@ -72,9 +67,16 @@ class PfsClient {
 
     delete(objectGetInfo, reqUids, callback) {
         const log = createLogger(reqUids);
-        logHelper(log, 'error', 'Not implemented', errors.NotImplemented,
+        const key = typeof objectGetInfo === 'string' ? objectGetInfo :
```

Comment:
If not a string, is `objectGetInfo` guaranteed to be an object?

Reply:
Well, it's javascript 😅, but it seems like it is. It's usually created by a `toObjectGetInfo()` function, and for example. Azure handles it exactly the same way -> https://github.com/scality/cloudserver/blob/development/8.1/lib/data/external/AzureClient.js#L226

Reply:
I see, LGTM. Just wanted to confirm that. 😄